src/main/java/frc/trigon/robot/constants/FieldConstants.java

```diff
@@ -6,13 +6,25 @@
 import edu.wpi.first.math.geometry.Pose3d;
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.geometry.Translation3d;
+import org.littletonrobotics.junction.Logger;
 import org.trigon.utilities.mirrorable.MirrorablePose2d;
 import org.trigon.utilities.mirrorable.MirrorableTranslation3d;
 
+import java.io.IOException;
 import java.util.HashMap;
 
 public class FieldConstants {
-    public static final AprilTagFieldLayout APRIL_TAG_FIELD_LAYOUT = AprilTagFields.k2024Crescendo.loadAprilTagLayoutField();
+        public static final AprilTagFieldLayout APRIL_TAG_FIELD_LAYOUT = AprilTagFields.k2024Crescendo.loadAprilTagLayoutField();
+//    public static final AprilTagFieldLayout APRIL_TAG_FIELD_LAYOUT;
+//
+//    static {
+//        try {
+//            APRIL_TAG_FIELD_LAYOUT = AprilTagFieldLayout.loadFromResource("2024-crescendo.json");
+//        } catch (IOException e) {
+//            throw new RuntimeException(e);
+//        }
+//    }
+
```

Comment:
This looks bad. Every year we'll need a home layout and the normal layout, find a good way to do it